htdocs/fourn/commande/card.php

```diff
@@ -405,7 +405,16 @@
 	}
 
 	// Add a product line
-	if ($action == 'addline' && $usercancreate) {
+	if ($action == 'addline' && GETPOST('submitforalllines', 'aZ09') && GETPOST('vatforalllines', 'alpha')) {
```

Comment:
We are missing a test on permission here.
May be    && $usercancreate

Reply:
Corrected

Reply:
Ok, fixed